cluster/examples/kubernetes/ceph/object-bucket-claim-delete.yaml

```diff
@@ -11,3 +11,7 @@ spec:
   #bucketName: 
   generateBucketName: ceph-bkt
   storageClassName: rook-ceph-delete-bucket
+  #additionalConfig:
```

Comment:
Is there any impact to having this uncommented by default? The individual settings would just be commented.
```suggestion
  additionalConfig:
```

Reply:
Sure @jeffvance recommended this way in my previous patch set

Reply:
It's fine to leave the stanza name un-commented and comment out the values.